google_takeout_parser/path_dispatch.py

```diff
@@ -52,17 +52,20 @@
     "YouTube and YouTube Music/subscriptions": None,  # TODO: parse
     "YouTube and YouTube Music/videos": None,
     "Location History/Semantic Location History": None,  # not that much data here. maybe parse it?
-    "Location History/Location History.json": _parse_location_history,  # old path to Location History
+    # optional space to handle pre-2017 data
+    "Location History/Location( )?History.json": _parse_location_history,  # old path to Location History
     "Location History/Records.json": _parse_location_history,  # new path to Location History
     "Location History/Settings.json": None,
     # HTML/JSON activity-like files which aren't in 'My Activity'
-    "YouTube and YouTube Music/history/.*?.html": _parse_html_activity,
-    "YouTube and YouTube Music/history/.*?.json": _parse_json_activity,
+
+    # optional " and Youtube Music" to handle pre-2017 data
+    r"YouTube( and YouTube Music)?/history/.*?.html": _parse_html_activity,
+    r"YouTube( and YouTube Music)?/history/.*?.json": _parse_json_activity,
```

Comment:
added 'r' here, even if it's not strictly necessary, imo much easier to read/spot typos when it's vertically aligned

Reply:
yeah, should probably use them for all of these -- thanks